client/js/components/assessments/universal_input.jsx

```diff
@@ -165,12 +165,24 @@ export default class UniversalInput extends React.Component{
           />
         break;
       case "movable_object_chain":
+        var selectAnswer = _.curryRight(props.selectAnswer);
```

Comment:
FP style is typically to actually provided the arguments to be curried when invoking curry:

```var selectAnswer = _.curryRight(props.selectAnswer)(false);```